mychevy/mychevy.py

```diff
@@ -110,7 +111,7 @@ def from_json(self, data):
             for a in CAR_ATTRS:
                 setattr(self, a, d[a])
 
-        except json.JSONDecodeError:
+        except ValueError:
```

Comment:
Yeh, JSONDecodeError was added in python 3.5, we can roll it back to value error for earlier versions. Probably also add 3.4 back into travis testing.

Reply:
Looking at the Travis tests I wasn't aware that this was developed with Python3. I have been doing all of my development and runs with Python2. Since Python2 isn't tested it might be nice to mention Python3 either in the README or put a `sys.version_info[0]` check in there to warn users that Python2 is unsupported.

Reply:
Ah, good idea. I had the trove information up to date, but we could throw a warning there as well.